docs/preview.rst

```diff
@@ -0,0 +1,7 @@
+Tech previews
+=============
+
+The following features are currently being released as part of a tech preview.
+
+* Distribution (Kickstart) Trees support
+* Advisory upload
```

Comment:
i would add more info here, that it is possible to upload advisory by providing a file or href of already existing artifact in pulp.